test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -3938,6 +3939,7 @@ void parsesFurnitureDiscovery() {
             "Round 2: Gog spots a sous vide laboratory inside the garbage disposal and runs out of his condo. He drags it back to the condo and stores it in the attic.";
         assertThat(text, containsString(expected));
         assertThat("leprecondoDiscovered", isSetTo("1,13,21"));
+        assertThat("_leprecondoFurniture", isSetTo(2));
```

Comment:
Ha ha. My draft PR adds a line in this exact spot.
I'll approve this and then resolve my conflict once this is merged. :)